### PR TITLE
Deposit Result Search with stored

### LINKED ISSUE #14: Deposit panel: Result of search tracking number
- Deposits should have a button to mark a shipment as stored in their deposit
- After a deposit marked a shipment as stored should be saved in the shipment history it was on the deposit city and country and the date.

### PR DESCRIPTION
**Pull request by cdsandoval**
Close #14 
Deposits should have a button to mark a shipment as stored in their deposit
After a deposit marked a shipment as stored should be saved in the shipment history it was on the deposit city and country and the date.

### CODE REVIEW

File: app/controllers/shipments_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class ShipmentsController < ApplicationController
-
+
```
Comment (Linzeur) [inline app/controllers/shipments_controller.rb]: Please, you could delete that so this file don't show as changed file

File: app/models/shipment.rb [hunk 2]
```diff
@@ -6,4 +6,11 @@ class Shipment < ApplicationRecord
   validates :origin_address, :destination_address, presence: true
   validates :weight, :freight_value, numericality: { greater_than: 0 }
 
+  def validate_stored?(user)
+    history = self.shipment_locations.last
+    history &&
+    history.city == user.city &&
+    history.country == user.country
```
Comment (Linzeur) [inline app/models/shipment.rb]: I like it as you split the conditionals where you put one conditional per line

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <h1>Welcome to FedEx Clone</h1>
 
-<h2>Find your shipment</h2>
+<h2>Find your shipments</h2>
```
Comment (Linzeur) [inline app/views/home/index.html.erb]: I think that it is not necessary add the character "s" in this view, you could undo that

File: app/controllers/shipments_controller.rb (hunk 1 above)
Comment (cdsandoval) [reply app/controllers/shipments_controller.rb]: Sure. I already did it.

File: app/models/shipment.rb (hunk 2 above)
Comment (cdsandoval) [reply app/models/shipment.rb]: It was a good idea of my technical lead :1st_place_medal:

File: app/views/home/index.html.erb (hunk 3 above)
Comment (cdsandoval) [reply app/views/home/index.html.erb]: I was testing the view, thanks for let me know it.

**Comment by cdsandoval** [comment]
> I realize that you haven't added the authorization created times ago, please you will add

Yes, sir. Just added it :+1:

File: app/models/shipment.rb (hunk 2 above)
Comment (yummta) [reply app/models/shipment.rb]: You're welcome.

**Pull request merged**